ui/app/models/task-group-scale.js

```diff
@@ -17,7 +17,7 @@ export default class TaskGroupScale extends Fragment {
   @fragmentArray('scale-event') events;
```

Comment:
We do this elsewhere, I just forgot to 😳 

```suggestion
  @fragmentArray('scale-event', { defaultValue: () => [] }) events;
```